mmv1/products/privilegedaccessmanager/Settings.yaml

```diff
@@ -0,0 +1,263 @@
+  - 'projects/{project}/locations/{location}/settings'
+description: |
+  Settings resource defines the properties, applied directly to the resource or inherited through the hierarchy, to enable consistent, federated use of PAM.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/iam/docs/pam-overview'
+    'Configure Privileged Access Manager settings': 'https://cloud.google.com/iam/docs/pam-configure-settings'
+  api: 'https://cloud.google.com/iam/docs/reference/pam/rest'
+docs:
+id_format: '{{parent}}/locations/{{location}}/settings'
+base_url: '{{parent}}/locations/{{location}}/settings'
+self_link: '{{parent}}/locations/{{location}}/settings'
+create_url: '{{parent}}/locations/{{location}}/settings'
+exclude_delete: true
+update_verb: 'PATCH'
+create_verb: 'PATCH'
+update_mask: true
+custom_code:
+  pre_update: 'templates/terraform/pre_update/privileged_access_manager_settings.go.tmpl' # get etag
+  custom_create: 'templates/terraform/custom_create/privileged_access_manager_settings.go.tmpl'
+examples:
+  - name: 'privileged_access_manager_settings_basic'
+    primary_resource_id: 'settings'
+    test_env_vars:
+      project: 'PROJECT_NAME'
+    exclude_test: true
+import_format:
+  - '{{%parent}}/locations/{{location}}/settings'
+min_version: 'beta'
+autogen_async: true
+async:
+  actions: ['create', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+  result:
+    resource_inside_response: true
+parameters:
+  - name: 'location'
+    type: String
+    description: |
+      The region of the PAM settings resource.
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'parent'
+    type: String
+    description: |
+      Format: projects/{project-id|project-number} or organizations/{organization-number} or folders/{folder-number}
+    url_param_only: true
+    required: true
+    immutable: true
+properties:
+  - name: 'name'
+    type: String
+    description: |
+      Name of the settings resource. Possible formats:
+      projects/{project-id|project-number}/locations/{location}/settings folders/{folder-number}/locations/{location}/settings organizations/{organization-number}/locations/{location}/settings
+    output: true
+  - name: 'createTime'
+    type: String
+    description: |
+      Create timestamp. Uses RFC 3339, where generated output will always be Z-normalized and use 0, 3, 6 or 9 fractional digits. Offsets other than "Z" are also accepted.
+      Examples: "2014-10-02T15:01:23Z", "2014-10-02T15:01:23.045123456Z" or "2014-10-02T15:01:23+05:30".
+    output: true
+  - name: 'updateTime'
+    type: String
+    description: |
+      Update timestamp. Uses RFC 3339, where generated output will always be Z-normalized and use 0, 3, 6 or 9 fractional digits. Offsets other than "Z" are also accepted.
+      Examples: "2014-10-02T15:01:23Z", "2014-10-02T15:01:23.045123456Z" or "2014-10-02T15:01:23+05:30".
+    output: true
+  - name: 'etag'
+    type: String
```

Comment:
Let's change the type to Fingerprint? IMO that is more suited for etag. Is there any specific reason you kept it String?